documentation/modules/deploying/proc-deploy-topic-operator-standalone.adoc

```diff
@@ -15,6 +15,11 @@ Standalone deployment files are provided with Strimzi.
 Use the `05-Deployment-strimzi-topic-operator.yaml` deployment file to deploy the Topic Operator.
 Add or set the environment variables needed to make a connection to a Kafka cluster.
 
+The Topic Operator watches for `KafkaTopic` resources in the namespace of a Kafka cluster deployment.
+You specify a namespace and connection to the Kafka cluster to watch in the configuration.
+The Topic Operator can only watch a single Kafka cluster in a namespace.
+If you want to use the operator with more than one Kafka cluster, install the clusters to separate namespaces.
```

Comment:
I don't think this makes sense for the standalone deployment.

```suggestion
The Topic Operator watches for `KafkaTopic` resources in a single namespace.
You specify the namespace and connection to the Kafka cluster to watch in the configuration.
A single namespace should be watched only by a single Topic Operator.
If you want to use multiple Topic Operators with different Kafka clusters, they should watch different namespaces.
```

Reply:
Changed to:

TO

> The Topic Operator watches for `KafkaTopic` resources in a single namespace.
> You specify the namespace to watch, and the connection to the Kafka cluster, in the Topic Operator configuration.
> A single Topic Operator can watch a single namespace. 
> Use more than one Topic Operator to watch different namespaces. 
> In this way, you can use the Topic Operator with multiple Kafka clusters.

CO

> The User Operator watches for `KafkaUser` resources in a single namespace.
> You specify the namespace to watch, and the connection to the Kafka cluster, in the User Operator configuration.
> A single User Operator can watch a single namespace. 
> Use more than one User Operator to watch different namespaces. 
> In this way, you can use the User Operator with multiple Kafka clusters.